### ISSUE TITLE
Emitted value instead of an instance of Error

### ISSUE THREAD

**Issue by burnmaniac** [trendmicro-frontend/react-dropdown#11]
Hello.

I am having this warning in the console, if anyone could check it out. I might send a PR, but not at this time.

```./node_modules/@trendmicro/react-dropdown/dist/react-dropdown.css
(Emitted value instead of an instance of Error) autoprefixer: /node_modules/@trendmicro/react-dropdown/dist/react-dropdown.css:172:3: You should write display: flex by final spec instead of display: box

**Comment by cheton** [comment]
It's a vendor prefix for backward compatibility with IE browsers. This CSS file is already prefixed, you should not apply autoprefixer for the imported CSS file.

**Comment by burnmaniac** [comment]
Well, I am using Create React App and I am importing CSS file for dropdown as stated in the example in the tutorial. I didn't think of that, so thank you for pointing out what's the problem.

Can you suggest any fixes? I don't have any other file to import instead of this already prefixed file.

**Comment by cheton** [comment]
I don't know if there is a way to disable autoprefixer only for specific files. For a workaround way, you can copy the CSS file to your project folder, remove the line of "display: box", and import it locally.

**Comment by burnmaniac** [comment]
I am not sure either. Couldn't find anything on Google.

However, I looked a bit deeper in the warning and found something. I don't think that this has anything to do with file being prefixed multiple times. It has to do something with final specification for the display property.

I guess you should update autoprefixer to the latest version and run build again.

Please, have a look at this: https://github.com/postcss/autoprefixer#warnings

Edit: I forgot to mention that I didn't see this warning in 1.2.0 version.

**Comment by cheton** [comment]
Fixed in v1.2.2
